### PR TITLE
Add about page

### LINKED ISSUE #5: About page
As a possible learner, i should see the about page to understand the project goals

### PR DESCRIPTION
**Pull request by aureleoules**
## Pull Request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build was run locally and any changes were pushed
- [x] Lint has passed locally and any fixes were made for failures

## Pull Request type

Please check the type of change your PR introduces:

- [ ] :bug: Bugfix
- [x] :sparkles: Feature
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :wrench: Other (please describe):

This PR adds the About Page.
The fonts used are incorrect but I don't think it is related to this PR.

Desktop
image

Mobile
image

Closes #5.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: app/about/page.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+          <h3 className="text-4xl">How to contribute</h3>
+          <p className="text-xl text-justify pt-4 pb-20">
+            Open an issue or a PR on <a target="_blank" rel="noreferrer" href="https://github.com/saving-satoshi/saving-satoshi">GitHub</a>. All suggestions are welcome,
+            including content changes, game mechanics – really anything.
+          </p>
+
+          <h3 className="text-4xl">Enjoy!</h3>
+          <p className="text-xl text-justify pt-4 pb-20">
+            Contributions by @satsie, @adamcjonas, @gbks, ... add other
```
Comment (adamjonas) [inline app/about/page.tsx:54]: can change post merge but these should be links to twitter and should add @_jsms_

**Comment by adamjonas** [comment]
ACK [1ab8707](https://github.com/saving-satoshi/saving-satoshi/pull/14/commits/1ab8707c0c9524b8abd11c49e6c46546797d7e91)

There is going to be a need for copyedit follow-ups but ran locally and this is better than a 404

File: app/about/page.tsx (hunk 1 above)
Comment (satsie) [reply app/about/page.tsx:54]: I am going to make a follow up PR for some updates. Will add JSMS (and John and Aurèle)

**Review by satsie — APPROVED** [review]
I will be making a follow up PR that builds on this. +1 to Jonas's comment about this being better than a 404. Merging.

**Comment by satsie** [comment]
Follow up PR: https://github.com/saving-satoshi/saving-satoshi/pull/25

**Pull request merged**